cli/pkg/oauth/oauth.go

```diff
@@ -58,7 +59,7 @@ func (s *OAuthServer) GetAuthJWT() error {
 		return fmt.Errorf("failed to start oauth server: %w", err)
 	}
 
-	loginUrl := fmt.Sprintf("%soauth?callback=%s", s.frontendEndpoint, url)
+	loginUrl := path.Join(s.frontendEndpoint, fmt.Sprintf("oauth?callback=%s", url))
```

Comment:
```suggestion
	loginUrl := url.JoinPath(s.frontendEndpoint, fmt.Sprintf("oauth?callback=%s", url))
```

Reply:
I had to create an alias for `url` as there is already an `url` var in this scope.

`neturl net/url`